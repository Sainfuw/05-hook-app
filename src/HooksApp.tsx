import React from "react";
import { MainApp } from "./components/09-useContext/MainApp";
// import { TodoApp } from "./components/08-useReducer/TodoApp";
// import { Padre } from "./components/07-tarea-memo/Padre";
// import { CallbackHook } from "./components/06-memos/CallbackHook";
// import { MemoHook } from "./components/06-memos/MemoHook";
// import { Memorize } from "./components/06-memos/Memorize";
// import { Layout } from "./components/05-useLayoutEffect/Layout";
// import { RealExampleRef } from "./components/04-useRef/RealExampleRef";
// import { MultipleCustomHooks } from "./components/03-Examples/MultipleCustomHooks";
// import { CounterApp } from "./components/01-useState/CounterApp";
// import { CounterWithCustomHook } from "./components/01-useState/CounterWithCustomHook";
// import { SimpleForm } from "./components/02-useEffect/SimpleForm";
// import { FormWithCustomHook } from "./components/02-useEffect/FormWithCustomHook";
// import { FocusScreen } from "./components/04-useRef/FocusScreen";

export const HooksApp = () => {
  return (
    <>
      {/* <CounterApp /> */}
      {/* <CounterWithCustomHook /> */}
      {/* <SimpleForm /> */}
      {/* <FormWithCustomHook /> */}
      {/* <MultipleCustomHooks /> */}
      {/* <FocusScreen /> */}
      {/* <RealExampleRef /> */}
      {/* <Layout /> */}
      {/* <Memorize /> */}
      {/* <MemoHook /> */}
      {/* <CallbackHook /> */}
      {/* <Padre /> */}
      {/* <TodoApp /> */}
      <MainApp />
    </>
  );
};
