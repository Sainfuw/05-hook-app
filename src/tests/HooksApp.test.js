import "@testing-library/jest-dom";
import { shallow } from "enzyme";
import { HooksApp } from "../HooksApp";

describe("Pruebas en HooksApp.test", () => {
  test("comprobar que el componente se vea correctamente", () => {
    const wrapper = shallow(<HooksApp />);
    expect(wrapper).toMatchSnapshot();
  });
});
