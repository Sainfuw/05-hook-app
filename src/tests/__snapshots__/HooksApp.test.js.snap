// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en HooksApp.test comprobar que el componente se vea correctamente 1`] = `
<Fragment>
  <MainApp />
</Fragment>
`;
